### ISSUE TITLE
Client crashes when cycling to silent gear/gems recipes on crafting bench

### ISSUE THREAD

**Issue by stillg0ld** [SilentChaos512/Silent-Gear#61]
## Versions

- **Silent's Gems**: 1.12.2-2.8.18+ 322
- **Silent Gear**: 1.12.2-0.6.4+ 72
- **Silent Lib**: 1.12.2-3.0.13+ 167 
- **Forge**:  1.12.2-14.23.5.2847

## Expected Behavior

Using crafting table with craftingtweaks installed, crafting recipe should cycle through recipes and let me select silentgear recipes.

- (Description)

## Actual Behavior

When loading crafting table with required ingredients (4 iron ingots for boots in this case), I get the original minecraft recipe with the arrow to cycle to silent gear's recipe. When I click the arrow to cycle, the client crashes immediately. The strangest thing is that before at random times, when opening the minecraft crafting table or silent gear's crafting station, I would get roughly the same error on my serverside gui with no crash. If this happens again, I will add the log to this ticket.

- (Description)

## Links/Images

Clientside crash report attached to ticket

- (Link)

## Steps to Reproduce the Problem

Joined back in and a placed iron in crafting table as a shaped, boots recipe. Tried to cycle through and problem reproduced.
[crash-2019-09-12_13.46.48-client.txt](https://github.com/SilentChaos512/SilentGems/files/3610126/crash-2019-09-12_13.46.48-client.txt)

**Comment by SilentChaos512** [comment]
Possibly fixed in version 0.6.5

**Comment by stillg0ld** [comment]
Alright, i'll get on that now and let you know. Also, I did manage to reproduce the issue with the server error by just using a normal crafting table. It happened when drag+placing cobblestone in a vanilla crafting table for a furnace. Tried using a few other ingredients with no error, then tried the cobblestone again and got the same error. Attaching a screenshot of the log.
![Screenshot (26)](https://user-images.githubusercontent.com/55283257/64884835-a6663080-d630-11e9-9344-240cc411f8c3.png)
